### PR TITLE
Github pages and Discord verification

### PR DESCRIPTION
**Pull request by oliik2013**
this is for the github pages verification

### CODE REVIEW

**Comment by oliik2013** [comment]
oh i forgot .json extension

**Comment by oliik2013** [comment]
+ Added Discord verification

File: domains/_discord.oliik2013.is-a.dev.json
Comment (CuteDog5695) [inline domains/_discord.oliik2013.is-a.dev.json:1]: Remove .is-a.dev from the name

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/_discord.oliik2013.is-a.dev.json
Comment (oliik2013) [reply domains/_discord.oliik2013.is-a.dev.json:1]: ok

Comment (oliik2013) [reply domains/_discord.oliik2013.is-a.dev.json:1]: done

**Review by iostpa — APPROVED** [review]
Seems good to me now

**Pull request merged**